components/CandidateTestView/TextEditor/TextEditorContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {MutableRefObject, useRef, useState} from "react";
```

Comment:
This whole file differs from the version on master - https://github.com/techswitch-learners/testswitch-frontend-candidate/blob/master/components/TextEditor/TextEditorContainer.tsx

The one on master is really the one that we want to be using going forward I think.
Could you move any changes that we need to keep over to that version?